community-call/20240919.md

```diff
@@ -0,0 +1,209 @@
+---
+title: SCS Community Call
+type: slide
+tags: presentation
+slideOptions:
+  theme: white
+  transition: 'slide'
+  parallaxBackgroundSize: '1920px 1080px'
+---
+
+
+<style>
+    .slides h1 {
+        font-size: 32px;
+        font-family: lato;
+        color: "#50c3a5";
+    }
+    .slides h2 {
+        color: "#0f5fe1";
+        font-size: 28px;
+        font-family: lato;
+    }
+    .slides h3, .slides h4 {
+        font-size: 20px;
+        font-family: lato;
+    }
+    .slides li {
+        font-size: 16px;
+        font-family: lato;
+    }
+    .slides p {
+        font-size: 16px;
+        font-family: lato;
+    }
+    .slides ul {
+        display: block!important;
+    }
+</style>
+
+
+
+
+<!-- Ease preperation with the template: https://github.com/SovereignCloudStack/minutes/tree/main/community-call/template.md -->
+
+# <font color="#50c3a5" style="text-shadow: -1px 1px 0 #FFF, 1px 1px 0 #FFF, 1px -1px 0 #FFF, -1px -1px 0 #FFF;">Sovereign</font> <font color="#0f5fe1" style="text-shadow: -1px 1px 0 #FFF, 1px 1px 0 #FFF, 1px -1px 0 #FFF, -1px -1px 0 #FFF;">Cloud Stack</font>
+## <font color="#7D7D82" style="text-shadow: -1px 1px 0 #FFF, 1px 1px 0 #FFF, 1px -1px 0 #FFF, -1px -1px 0 #FFF;">One platform – standardized, built and operated by many.</font>
+## <font color="#0f5fe1" style="text-shadow: -1px 1px 0 #FFF, 1px 1px 0 #FFF, 1px -1px 0 #FFF, -1px -1px 0 #FFF;">Community Call 2024-09-19</font>
+
+![](https://input.scs.community/uploads/a46465b4-dd13-45ec-803c-b53912a9cae9.jpg)
+
+
+---
+
+![](https://input.scs.community/uploads/ae131de0-65bb-41b6-80b4-792a51f33961.png)
+
+https://en.wikipedia.org/wiki/International_Talk_Like_a_Pirate_Day
+
+---
+
+# Team IaaS
+
+We build the reference implementation of the IaaS parts of SCS 
+
+Report by <a href="https://github.com/mbuechse">@mbuechse</a>
+
+----
+
+## Team IaaS
+
+part 1/2
+
+- vPTG is upcoming: 2024-10-21 through -25 virtual
+- Update on Public-Cloud SIG: changed weeks
+- Lean Operator coffee next monday: talk standardization
+
+----
+
+## Team IaaS
+
+part 2/2
```

Comment:
```suggestion
part 2/3
```

Reply:
I want to add the slides in the way they were presented. Why adjust this?

Reply:
well, actually. I don't care :)